core/templates/pages/admin-page/misc-tab/admin-misc-tab.component.ts

```diff
@@ -62,7 +59,7 @@ export class AdminMiscTabComponent {
   MAX_USERNAME_LENGTH: number = AppConstants.MAX_USERNAME_LENGTH;
   message: string = '';
   expIdToGetInteractions!: string;
```

Comment:
interactions --> interactionIds

(and you probably want a "for" at the end -- expIdToGetInteractionIdsFor)

Reply:
Done.